enzyme/test/ActivityAnalysis/integration.ll

```diff
@@ -0,0 +1,105 @@
+; RUN: if [ %llvmver -lt 16 ]; then %opt < %s %loadEnzyme -enzyme -S | %llc -filetype=obj -o output.o && %clang -no-pie output.o -o output && ./output | FileCheck %s
```

Comment:
Can this be changed to be an activity analysis check [e.g. check that the print of the activity analysis is constant val/inst], like other tests in this folder?

A runtime integration test can separately be added to test/Integration/ReverseMode

Reply:
Yes I am adding that and removing the integration one